source/content/guides/integrated-composer/02-dependencies.md

```diff
@@ -12,7 +12,25 @@ permalink: docs/guides/integrated-composer/dependencies
 anchorid: dependencies
 ---
 
-This sections provides information on how to add or remove individual site dependencies.
+This sections provides information on dependency requirements and how to add or remove individual site dependencies.
+
+## Composer `require dev` and `require` Sections
+
+It's important to understand how `require dev` and `require` are used on the Pantheon platform.
+
+### Composer `require dev` Section
+
+You should use the `require-dev` section for dependencies that are not a part of the web application but are necessary to build or test your project. Some examples are `php_codesniffer` and `phpunit`. Dev dependencies are not be deployed to Pantheon.
+
+Third party dependencies, such as modules / plugins and themes, are added to the project via `composer.json`. The `composer.lock` file keeps track of the exact version of dependency. [Composer `installer-paths`](https://getcomposer.org/doc/faqs/how-do-i-install-a-package-to-a-custom-path-for-my-framework.md#how-do-i-install-a-package-to-a-custom-path-for-my-framework-) are used to ensure the dependencies are downloaded into the appropriate directory.
+
+Non-WordPress dependencies are downloaded to the `/vendor` directory.
+
+### Composer `require` Section
+
+You should use the `require` section of your `composer.json` file for dependencies your web project needs, even dependencies only used on non-Live environments. All dependencies in the `require` section are pushed to Pantheon. 
+
+You can use  `--no-dev` to avoid the presence of Dev dependencies in Live and Test environments.
```

Comment:
This is not really true. You can not decide this, Pantheon will never push "require-dev" to test/live but will always push it to dev/multidev